portal-web/test/functional/com/liferay/portalweb/paths/plugin/workflow/WorkflowMetrics.path

```diff
@@ -173,6 +183,16 @@
 	<td>//tr[contains(*, '${rowName}')]//td[@data-testid='slaBreached']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>METRICS_PERFORMANCE_BY_ASSIGNEE_DROPDOWN_STEP_REVIEW</td>
+	<td>//div[contains(@class, 'dropdown-menu')]//span[contains(text(),'Review')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>METRICS_PERFORMANCE_BY_ASSIGNEE_DROPDOWN_STEP_UPDATE</td>
+	<td>//div[contains(@class, 'dropdown-menu')]//span[contains(text(),'Update')]</td>
+	<td></td>
+</tr>
```

Comment:
Hi @Adriverforlife 
Are we using these locators? I didn't find any mention of these locators in the script

Reply:
I think I added those when I was trying to do something else and then ended up not using them. I can remove them.

Reply:
Actually, found where I used them. macro selectStepFilterPerformaceByAssignee